### DISCUSSION TITLE
Proposed scenario order change for Wacky Worlds

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#21545] [Feature requests]
After conducting a poll, then personally vetting the scenarios in collaboration with others on the discord, I have here a suggested new scenario order for the Wacky Worlds expansion pack:

Beginner Parks
Grand Canyon
Great Wall of China
African Diamond Mine
Ayers Rock
Rollercoaster Heaven

Challenging Parks
Oasis
Rio Carnival
Maharaja Palace
Victoria Falls
Extreme Hawaiian Island
Rain Forest Plateau
Renovation

Expert Parks
Ecological Salvage
European Cultural Festival
Fun at the Beach
Inca Lost City
Japanese Coastal Reclaim

**Comment by xrgtmx** [comment]
Is there an ongoing poll for Time Twister scenario order? I can add it to my current PR for scenario order.
